parsec/core/logged_core.py

```diff
@@ -40,6 +45,55 @@
 
 logger = get_logger()
 
+FAILSAFE_PATTERN_FILTER = re.compile(
+    r"^\b$"
+)  # Matches nothing (https://stackoverflow.com/a/2302992/2846140)
+
+
+def _get_pattern_filter(pattern_filter_path: Path) -> Optional[Pattern]:
+    try:
+        data = pattern_filter_path.read_text()
+    except OSError as exc:
+        logger.warning(
+            f"Path to the file containing the filename patterns "
+            f"to ignore is not properly defined: {exc}"
+        )
+        return None
+    try:
+        regex = "|".join(
```

Comment:
time to use a for loop I guess:
```python
regex = []
for line in data.splitlines():
    line = line.strip()
    if line and not line.startswith("#"):
        regex.append(fnmatch.translate(line))
regex = "|".join(regex)
```

Reply:
:+1:

Reply:
e55ed1ec6d0c085d18bfdc8c964be92f342860b4